tests/redux-persist_mock.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+jest.mock('redux-persist', () => ({
```

Comment:
This was previously causing a lot of noise in unit tests since it was causing a lot of null pointer errors. Now that it's mocked out, it won't do that any more